components/new-contribution-flow/ContributionFlowSuccess.js

```diff
@@ -232,10 +243,65 @@ class NewContributionFlowSuccess extends React.Component {
     return <PublicMessageForm order={this.props.data.order} publicMessage={publicMessage} />;
   };
 
+  renderBankTransferInformation = () => {
+    const bankTransferInstructions = get(
+      this.props.data,
+      'order.toAccount.host.settings.paymentMethods.manual.instructions',
+      null,
+    );
+
+    return (
+      <Flex flexDirection="column" justifyContent="center" width={[1, 3 / 4]} px={[4, 0]} py={[2, 0]}>
+        <MessageBox type="warning" fontSize="12px">
+          <FormattedMessage
+            id="collective.user.orderProcessing.manual"
+            defaultMessage="<i18n-bold>Your donation is pending.</i18n-bold> Please follow the instructions in the confirmation email to manually pay the host of the collective."
+            values={{
+              'i18n-bold': I18nBold,
+            }}
+          />
+        </MessageBox>
+        <BankTransferInfoContainer my={3} p={4}>
+          <H3>
+            <FormattedMessage id="NewContributionFlow.PaymentInstructions" defaultMessage="Payment instructions" />
+          </H3>
+          <Flex mt={2}>
+            <HTMLContent>{bankTransferInstructions}</HTMLContent>
+          </Flex>
+        </BankTransferInfoContainer>
+        <Flex px={3}>
+          <P fontSize="16px" color="black.700">
+            <FormattedMessage
+              id="NewContributionFlow.InTheMeantime"
+              defaultMessage="In the meantime, you can follow {collective} and see how they are spending the money {collectiveLink}."
+              values={{
+                collective: this.props.data.order.toAccount.name,
+                collectiveLink: (
+                  <Link
+                    route="collective"
+                    params={{
+                      slug: this.props.data.order.toAccount.slug,
+                    }}
+                  >
+                    <FormattedMessage
+                      id="NewContributionFlow.OnCollectivePage"
+                      defaultMessage="on their collective page"
+                    />
+                  </Link>
+                ),
+              }}
+            />
```

Comment:
This goes against https://docs.opencollective.com/help/contributing/development/translations#dont-assume-words-order-stays-the-same-in-other-languages. It's also not clear for contributors who to translate this, if you're given the string:
> In the meantime, you can follow {collective} and see how they are spending the money {collectiveLink}.

You have no clue that `collectiveLink` will be replaced by a full string (`on their collective page`). Similarly, you don't have the context when asked to translate `on their collective page`.

Suggested alternative:

```suggestion
            <FormattedMessage
              id="NewContributionFlow.InTheMeantime"
              defaultMessage="In the meantime, you can follow {collective} and see how they are spending the money <CollectiveLink>on their collective page</CollectivePage>."
              values={{
                collective: this.props.data.order.toAccount.name,
                CollectiveLink: getI18nLink({ as: CollectiveLink, collective: this.props.data.order.toAccount }),
              }}
            />
```